appveyor.yml

```diff
@@ -1,30 +1,34 @@
 build: false
-image: Visual Studio 2019
 shallow_clone: false
 skip_branch_with_pr: true
+image:
+  - MacOS
+  - Visual Studio 2019
+
 environment:
 
   global:
     PYTHONUNBUFFERED: "1"
     INSTALL_EDM_VERSION: "3.0.1"
+    APPVEYOR_YML_DISABLE_PS_LINUX: true
 
   matrix:
     - RUNTIME: '3.6'
-      TOOLKITS: "null"
+      TOOLKIT: "pyqt"
     - RUNTIME: '3.6'
-      TOOLKITS: "pyqt"
+      TOOLKIT: "pyqt5"
     - RUNTIME: '3.6'
-      TOOLKITS: "pyqt5"
+      TOOLKIT: "pyside2"
     - RUNTIME: '3.6'
-      TOOLKITS: "pyside2"
+      TOOLKIT: "wx"
     - RUNTIME: '3.6'
-      TOOLKITS: "wx"
+      TOOLKIT: "null"
```

Comment:
For future reference, is the change in order important?

Reply:
The `TOOLKITS` -> `TOOLKIT` change was because the matrix does not test multiple toolkits. 
The `wx` -> `null` is an artefact of the git diff. There was no change effectively, just the order of the matrix changed.

Reply:
Oh I see. Thank you!

Reply:
> Oh I see.

Hmmm taking it back... I was referring to the change in order from "null", "pyqt", "pyqt5", "pyside2", "wx" to "pyqt", "pyqt5", "pyside2", "wx", "null".
I guess this was not important, just wondering.

Reply:
There is not specific reason/thought from my side, it just how I made the changes (see the commit history).

In general the order of the matrix matters because of the fast-failure option. One would probably place the more fragile builds at the top so that the build cancels early.

Reply:
Cool. Thank you.